README.md

```diff
@@ -56,6 +56,49 @@ make serve
 
 This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
 
+## Building the API reference pages
+
+The API reference pages located in `content/en/docs/reference/kubernetes-api/ref` are built from the Swagger specification, using https://github.com/feloy/kubernetes-resources-reference.
```

Comment:
This line should change to `https://github.com/kubernetes-sigs/reference-docs/gen-resourcesdocs`?